### PR TITLE
FA-50 - Criar componente Slider com range

### PR DESCRIPTION
**Pull request by henriquearthur**
- Criado componente Range Slider para os filtros do Comparador;
- Adicionado filtro de Número de Ativos para o Comparador;

### CODE REVIEW

File: lib/core/presentation/stores/comparator_store.dart [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                reit.assetsAmount <= assetsAmountRange.last,
+          )
+          .toList();
+    }
+
+    return reits;
+  }
+
+  @action
+  void toggle(Filter filter) {
```
Comment (fernandoeeu) [inline lib/core/presentation/stores/comparator_store.dart]:
```suggestion
void toggleFilter(Filter filter) {
```

**Review by fernandoeeu — APPROVED** [review]

File: lib/core/presentation/stores/comparator_store.dart (hunk 1 above)
Comment (henriquearthur) [reply lib/core/presentation/stores/comparator_store.dart]: Já tinha feito essa mudança nas próximas PRs a serem enviadas. Mas podemos fazer logo aqui também.

**Comment by henriquearthur** [comment]
O `analyze` está falhando com esses 2 erros:

    info • The operand can't be null, so the condition is always true • lib/core/presentation/stores/comparator_store.dart:66:42 • unnecessary_null_comparison
    
    error • The method 'toggle' isn't defined for the type 'ComparatorStore' • lib/modules/comparator_settings/presentation/comparator_settings_page.dart:45:53 • undefined_method

**Comment by henriquearthur** [comment]
O `analyze` está falhando com esses 2 erros:

    info • The operand can't be null, so the condition is always true • lib/core/presentation/stores/comparator_store.dart:66:42 • unnecessary_null_comparison
    
    error • The method 'toggle' isn't defined for the type 'ComparatorStore' • lib/modules/comparator_settings/presentation/comparator_settings_page.dart:45:53 • undefined_method

@fernandoeeu

**Comment by fernandoeeu** [comment]
> O `analyze` está falhando com esses 2 erros:
> 
> ```
> info • The operand can't be null, so the condition is always true • lib/core/presentation/stores/comparator_store.dart:66:42 • unnecessary_null_comparison
> 
> error • The method 'toggle' isn't defined for the type 'ComparatorStore' • lib/modules/comparator_settings/presentation/comparator_settings_page.dart:45:53 • undefined_method
> ```
> 
> @fernandoeeu

feito

**Comment by henriquearthur** [comment]
LGTM

**Pull request merged**